pkgs/development/libraries/sundials/3.x.lapack.nix

```diff
@@ -0,0 +1,34 @@
+{ cmake, fetchurl, python, stdenv,
+  openblas, liblapack,
+  gfortran }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sundials";
+  version = "3.2.1";
+
+  src = fetchurl {
+    url = "https://computation.llnl.gov/projects/${pname}/download/${pname}-${version}.tar.gz";
+    sha256 = "0238r1qnwqz13wcjzfsbcfi8rfnlxcjjmxq2vpf2qf5jgablvna7";
+  };
+
+
+  cmakeFlags = [
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+    "-DSUNDIALS_INDEX_TYPE=int32_t"
+    "-DLAPACK_ENABLE=ON"
+    "-DLAPACK_LIBRARIES=[${liblapack}/lib/liblapack.dylib,${openblas}/lib/libopenblas.dylib,${gfortran.cc.lib}/lib/libgfortran.a]"
```

Comment:
`.dylib` is macos-specific - we can't set it like this unconditionally on all platforms - see https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/sql/mariadb/default.nix#L86 for an example (I didn't check yet if `LAPACK_LIBRARIES` is autodiscovered properly on linux so far)